modules/canbus/vehicle/gem/gem_controller.cc

```diff
@@ -408,8 +409,7 @@ void GemController::Steer(double angle) {
     return;
   }
   // convert to radians
-  const double real_angle =
-      params_.max_steer_angle() * angle / 100.0 * M_PI / 180.0;
+  const double real_angle = vehicle_params_.max_steer_angle() * angle / 100.0;
```

Comment:
the comment "conver to radians" does not match with the code.

Reply:
@startcode Will changle

Reply:
@startcode Done